tests/suites/caasadmission/admission.sh

```diff
@@ -0,0 +1,76 @@
+
+run_deploy_microk8s() {
+  echo
+
+  name=$1
```

Comment:
Just to keep things standardized

```suggestion
  name=${1}
```